chain/chunks/src/lib.rs

```diff
@@ -449,13 +449,29 @@ impl ShardsManager {
             || self.runtime_adapter.will_care_about_shard(account_id, parent_hash, shard_id, is_me)
     }
 
+    /// Send requests for a chunk
+    /// Depending on whether the node tracks the shard, it either downloads the full chunk
+    /// or only parts that it concerns, specifically, the part that should be owned by the node if
+    /// it is a validator, plus the parts that are sent to the shards that the node is tracking
+    ///
+    /// `height`: block height of the chunk
+    /// `parent_hash`: hash of the prev block of the block that includes the requested chunk
+    /// `shard_id`: shard id of the requested chunk
+    /// `chunk_hash`: chunk hash of the requested chunk
+    /// `force_request_full`: if it is true, the node will request the full chunk regardless whether
+    ///    it is tracking the shard
+    /// `request_from_single_source`: if it is true, the node requests all parts from a single node
+    ///                               instead of sending one request to each part owner
+    /// `request_own_parts_from_others`: whether we should request parts owned by us
```

Comment:
this is the part I don't understand - what does it mean 'owned'

Reply:
Each part has an assigned owner (a validator). Basically an encoded chunk is divided into n parts where n is the number of validators and each validator get a partial encoded chunk